public/plugin.ts

```diff
@@ -9,21 +9,35 @@ import {
   CoreStart,
   Plugin,
 } from '../../../src/core/public';
-import {
-  FlowFrameworkDashboardsPluginSetup,
-  FlowFrameworkDashboardsPluginStart,
-} from './types';
+import { FlowFrameworkDashboardsPluginStart } from './types';
 import { PLUGIN_ID } from '../common';
-import { setCore, setRouteService } from './services';
+import {
+  setCore,
+  setRouteService,
+  setSavedObjectsClient,
+  setDataSourceManagementPlugin,
+  setDataSourceEnabled,
+  setNotifications,
+} from './services';
 import { configureRoutes } from './route_service';
+import { DataSourceManagementPluginSetup } from '../../../src/plugins/data_source_management/public';
+import { DataSourcePluginSetup } from '../../../src/plugins/data_source/public';
+
+export interface FlowFrameworkDashboardsSetupDeps {
+  dataSourceManagement: DataSourceManagementPluginSetup;
+  dataSource: DataSourcePluginSetup;
+}
```

Comment:
Can this be moved back into `public/types.ts` and keep the original name of `FlowFrameworkDashboardsPluginSetup` for consistency?